app/editor/menus/block.tsx

```diff
@@ -111,6 +112,18 @@ export default function blockMenuItems(dictionary: Dictionary): MenuItem[] {
       icon: TableIcon,
       attrs: { rowsCount: 3, colsCount: 3 },
     },
+    {
+      name: "math_inline",
+      title: dictionary.mathInline,
+      icon: MathIcon,
+      keywords: "math latex",
+    },
+    {
+      name: "math_display",
```

Comment:
```suggestion
      name: "math_block",
```

Ideally we'd name this `math_block` to match `code_block`